__test__/port.test.js

```diff
@@ -0,0 +1,42 @@
+const Port = require('../src/port.js')
+
+describe('port constructor', () => {
+    describe('has ports and ships', () => {
```

Comment:
Don't need to put these tests inside 2 describe blocks. Describe blocks are used to separate out different test cases, but as all your tests are inside of both of these you only need 1 describe block